lnrpc/lightning.proto

```diff
@@ -2891,6 +2891,10 @@ message QueryRoutesRequest {
     */
     repeated lnrpc.RouteHint route_hints = 16;
 
+    // An optional blinded path(s) to reach the destination. Note that the
+    // introduction node must be provided as the first hop in the route.
+    repeated BlindedPayment blinded_path = 19;
```

Comment:
wait so we _are_ adding support for path finding to multiple blinded routes here? or am i missing something?

Reply:
the naming is also a bit confusing. Since this is called `BlindedPath` but is an array of `BlindedPayments` each of which include a `BlindedPath`

Reply:
> the naming is also a bit confusing. Since this is called BlindedPath but is an array of BlindedPayments each of which include a BlindedPath

naming 🙇‍♀️

Reasoning here was: 
- `BlindedPayment`: this _isn't_ a blinded path, since blinded paths can exist without payment params 
- `blinded_path`: calling it `blinded_payments` feels like we're making multiple payments which is also wrong?

Going to go for `repeated BlindedPaymentPath blinded_payment_paths` and pray for my line wrapping?